app/views/ReportView/components/RapidSummary/index.tsx

```diff
@@ -386,8 +385,7 @@ const RapidSummary = ({
         value: msiStatus,
       },
     ]);
-  }, [microbial, primaryBurden, tmburMutBur, report.m1m2Score, report.sampleInfo, report.tumourContent, tCellCd8.percentile, tCellCd8.score, report.captiv8Score,
-    tCellCd8.percentileHidden, tCellCd8, tCellCd8.pedsScoreComment, tmburMutBur.adjustedTmb, tmburMutBur.tmbHidden, tCellCd8.pedsScore, tCellCd8.pedsPercentile]);
+  }, [microbial, primaryBurden, tmburMutBur, tCellCd8, report.m1m2Score, report.sampleInfo, report.tumourContent, report.captiv8Score]);
```

Comment:
I don't think we can remove the specific properties of tCellCd8 and tmburMutBur in the dependencies. Let's add '?' to validate the properties like this: [microbial, primaryBurden, tmburMutBur, report.m1m2Score, report.sampleInfo, report.tumourContent, tCellCd8?.percentile, tCellCd8?.score, report.captiv8Score,
    tCellCd8?.percentileHidden, tCellCd8, tCellCd8?.pedsScoreComment, tmburMutBur?.adjustedTmb, tmburMutBur?.tmbHidden, tCellCd8?.pedsScore, tCellCd8?.pedsPercentile]